zerver/views/streams.py

```diff
@@ -322,7 +322,7 @@ def add_subscriptions_backend(
     elif user_profile.realm.is_zephyr_mirror_realm:
         subscribers = set([user_profile])
     else:
-        subscribers = set()
+        return json_error(_("Stream must have at least one subscriber."))
```

Comment:
@YJDave: I think this is making `./tools/test-api` fail. Could you please investigate? :)